### PR TITLE
Bug: Docket QC form not retaining secondary document type in select

### PR DESCRIPTION
**Pull request by rachaelparris**
* Added secondaryDocument to Document entity with validation

Continuation of #5624 

https://trello.com/c/vhDUQQhe/560-docket-qc-form-not-retaining-document-type-for-secondary-document

### CODE REVIEW

File: web-client/src/presenter/computeds/fileDocumentHelper.js [hunk 1]
```diff
@@ -48,6 +48,9 @@ export const fileDocumentHelper = (get, applicationContext) => {
       .formatDateString(certificateOfServiceDate, 'MMDDYY');
   }
 
+  const showSecondaryDocument =
+    form.secondaryDocument && form.secondaryDocument.documentTitle;
```
Comment (matthopson) [inline web-client/src/presenter/computeds/fileDocumentHelper.js:52]: I struggle a little with knowing if I can use `documentTitle` vs `documentType`. Is this safe here?

**Review by rachelschneiderman — APPROVED** [review]

**Review by matthopson — APPROVED** [review]
One question that is probably not a concern (based on the previous code), otherwise 👍

File: web-client/src/presenter/computeds/fileDocumentHelper.js (hunk 1 above)
Comment (rachaelparris) [reply web-client/src/presenter/computeds/fileDocumentHelper.js:52]: I just pulled this logic out of the view. This is what was being used before, so I'm guessing it's okay here.

Comment (matthopson) [reply web-client/src/presenter/computeds/fileDocumentHelper.js:52]: Yeah, that was what I had assumed. I just know that `documentTitle` is optional and isn't on every document. Many times we have to do a `documentTitle || documentType` thing for this.

**Pull request merged**